editor.planx.uk/src/@planx/components/shared/Disclaimer.tsx

```diff
@@ -4,11 +4,24 @@ import { WarningContainer } from "@planx/components/shared/Preview/WarningContai
 import React from "react";
 import ReactMarkdownOrHtml from "ui/shared/ReactMarkdownOrHtml/ReactMarkdownOrHtml";
 
-export const Disclaimer = ({ text }: { text: string }) => (
-  <WarningContainer>
-    <ErrorOutline />
-    <Typography variant="body2" component="div" ml={2} sx={{ "& p:first-of-type": { marginTop: 0, } }}>
-      <ReactMarkdownOrHtml source={text} openLinksOnNewTab />
-    </Typography>
-  </WarningContainer>
-);
+export const Disclaimer = ({
+  text,
+  defaultText,
+}: {
+  text: string;
+  defaultText?: string;
```

Comment:
nit: see first comment - I'd personally favor a single `text` prop here that's always required & handle the `||` fallback condition at point of implementation!